docs/Testing.md

```diff
@@ -0,0 +1,124 @@
+SGX-LKL Testing
+===============
+
+How to run LTP tests
+--------------------
+
+To be able to run one or all LTP tests, we must first build sgx-lkl. Below are the step by step instructions to run all LTP tests or one LTP test locally.
+## Build and install SGX-LKL
```

Comment:
@letmaik  I checked the README.md for installaltion section and it looks very long and complicated. In my instructions it is much simpler and cleaner since using scripts. Should I submit a PR to change REDAME.md for sgx-lkl instructions? I will remove installation instructions here and refer to ../README.md

Reply:
Removed my build and install instructions. Referred to README.md 
We can submit a separate PR for simplifying build/install part of README.md using scripts. Out of scope of this PR.

Reply:
@letmaik any comment about simplifying README.md install instructions using scripts?

Reply:
Let's leave this as it will be easier once we got CMake.